data/content-script.js

```diff
@@ -2,6 +2,7 @@ function focus_code_field() {
     for (let input of document.querySelectorAll('[id^=duo-verify-code-]')) {
         if (input.offsetParent) {
             input.focus();
+            input.select();
```

Comment:
I added the select() to the focus function with the thinking that if there is a value in the field in any of the other cases you focus (after picking device, on page load with no devices) it will be an old value and also appropriate to over-write.